test/unit/local/steps/initial_diff.js

```diff
@@ -92,8 +92,8 @@ describe('local/steps/initialDiff', () => {
 
     const events = await buffer.pop()
     should(events).deepEqual([
-      builders.event().action('deleted').kind('file').path('bar').build(),
-      builders.event().action('deleted').kind('directory').path('foo').build(),
+      builders.event().action('deleted').kind('file').path('bar').noStats().build(),
+      builders.event().action('deleted').kind('directory').path('foo').noStats().build(),
```

Comment:
Shouldn't `deleted` events always have no stats?

Reply:
Maybe?? I can't really tell. But this seems plausible.

Reply:
Please note I actually don't know the answer :smile:

Reply:
Look at https://raw.githubusercontent.com/cozy-labs/cozy-desktop/master/doc/developer/win_watcher.png & https://raw.githubusercontent.com/cozy-labs/cozy-desktop/master/doc/developer/linux_watcher.png : there are no stats on the deleted lines.